protocol/str.go

```diff
@@ -19,7 +19,7 @@ func NewDirSTR(str *merkletree.SignedTreeRoot) *DirSTR {
 
 // Serialize overrides merkletree.SignedTreeRoot.Serialize
 func (str *DirSTR) Serialize() []byte {
-	return append(str.SerializeInternal(), str.Policies.Serialize()...)
+	return append(str.SignedTreeRoot.SerializeInternal(), str.Policies.Serialize()...)
```

Comment:
Why did you make this change? I think it can be simplified to `str.SerializeInternal()`.